tests/test_capture.py

```diff
@@ -284,3 +290,184 @@ def test_get_frequency(
     new_divider = 2
     monitoring.set_frequency(new_divider)
     assert mc.capture.get_frequency(servo=alias) == max_frequency / new_divider
+
+
+@pytest.mark.parametrize(
+    "name, axis",
+    [("CL_CUR_Q_SET_POINT", "1"), (1, 1)],
+)
+@pytest.mark.smoke
```

Comment:
smoke mark is not necessary because tests with virtual drive don't depend on the TESTS parameter in the pipeline.

Reply:
Done